util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java

```diff
@@ -102,6 +107,10 @@ public String getPermissionPropagationCheckboxLabel() {
 		return _permissionPropagationCheckboxLabel;
 	}
 
+	public boolean isPermissionPropagationCheckboxValueChecked() {
```

Comment:
I think isPermissionPropagationCheckboxValue is enough, just like other naming convention.

Reply:
Fixed it. Thanks